### DISCUSSION TITLE
@wpengine/headless Next 12 support.

### DISCUSSION THREAD

**Discussion by cschratz** [wpengine/faustjs#899] [Q&A] [answered]
Just tossing this question out to hopefully confirm or deny my suspicions. Currently working with a headless WP site that is utilizing @wpengine/headless to pull the WP data into the Next app. I bumped the Next version to 12 and I don't think @wpengine/headless can provide the functionality needed for Next 12. Am I correct or is my issue related to something else?

**Comment by blakewilson** [comment, marked as answer]
Hey @cschratz  👋 

I believe you are correct. The `@wpengine/headless` package is no longer supported in favor of the following packages:

* `@faustjs/core`
* `@faustjs/react`
* `@faustjs/next`

The following packages will support Next 12. For a more detailed look you can check out the [getting started example project](https://github.com/wpengine/faustjs/tree/canary/examples/next/getting-started).
